blockchain/event_filter.go

```diff
@@ -108,8 +110,12 @@ func (f *EventFilter) Events(cToken *ContinuationToken, chunkSize uint64) ([]*Fi
 			return nil, nil, err
 		}
 
-		// test `from` only by default because empty filter keys means match all
-		possiblyMatches := header.EventsBloom.Test(fromBytes[:])
+		// test `from` only by default if it's not nil because empty filter keys means match all
+		var possiblyMatches bool
```

Comment:
This is not quite the same thing. It breaks the “empty keys mean match all” behavior.

Reply:
I accounted for it [here](https://github.com/NethermindEth/juno/pull/761/files#diff-299bd13c9d85e40bab3e1c50f08f697257ed323a0db67685f628d71e3c635518R131)